apps/music/js/ui/views/player_view.js

```diff
@@ -300,6 +300,7 @@ var PlayerView = {
     } else {
       this.shuffleButton.classList.remove('shuffle-on');
     }
+    this.shuffleButton.setAttribute('aria-pressed', this.shuffleOption);
```

Comment:
As a rule, we try to separate ARIA from styling, and not use those attributes in CSS rules. This allows us the freedom to adjust the ARIA without risking changing layout.